tox.ini

```diff
@@ -1,6 +1,8 @@
 # content of: tox.ini , put in same dir as setup.py
 [tox]
 envlist=flake8,docs,py27,py26
+whitelist_externals =
```

Comment:
When running `docs`, tox tries to look for `make` in the env, and of course make isn't installed in the env. It still runs the docs, but displays a warning.
This change makes the warning go away :)